.eslintrc.json

```diff
@@ -149,6 +149,7 @@
               {
                 "sourceTag": "type:environments",
                 "onlyDependOnLibsWithTags": [
+                  "type:data-access",
```

Comment:
question: What do you think of removing this line in favor of adding `// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries` comment where the import to `import '@energinet-datahub/dh/shared/data-access-msw';` is inside `environment.ts` file? I think it's file to do it development.

Reply:
I think that's a very good solution!